hummingbot/cli/settings.py

```diff
@@ -153,9 +157,36 @@ def is_strategy(value: str) -> bool:
 
 
 def is_path(value: str) -> bool:
-    return isfile(value) and value.endswith('.yml')
+    return isfile(join(CONF_FILE_PATH, value)) and value.endswith('.yml')
+
+
+def is_valid_market_symbol(market: str, value: str) -> bool:
+    if symbol_fetcher.ready:
+        market_symbols = symbol_fetcher.symbols.get(market, [])
+        return value in symbol_fetcher.symbols.get(market) if len(market_symbols) > 0 else True
```

Comment:
no return if symbol_fetcher is not ready

Reply:
maybe add a comment explaining why default is true

Reply:
added